src/ecdsa/ecdsa.py

```diff
@@ -119,6 +119,13 @@ def __init__(self, generator, point):
     if point.x() < 0 or n <= point.x() or point.y() < 0 or n <= point.y():
       raise RuntimeError("Generator point has x or y out of range.")
 
+  def __eq__(self, other):
+    if isinstance(other, Public_key):    
+      """Return True if the points are identical, False otherwise."""
+      return self.curve == other.curve \
+        and self.point == other.point
+    return False
```

Comment:
I wonder if returning `NotImplemented`, like here:
https://github.com/warner/python-ecdsa/pull/127/files#diff-54eaad15d2c52f2d700a80792692c6b1L80-L82

wouldn't be a better idea; then alternative implementation could handle it, irrespective if the order of operands

Reply:
At least it makes more sense than just return False